core/domain/activity_jobs_one_off.py

```diff
@@ -33,17 +41,48 @@ class IndexAllActivitiesJobManager(jobs.BaseMapReduceOneOffJobManager):
 
     @classmethod
     def entity_classes_to_map_over(cls):
-        return [exp_models.ExpSummaryModel,
+        return [exploration_models.ExpSummaryModel,
```

Comment:
Any specific reason why you spelt out exploration_models instead of exp_models? I think the rest of the codebase uses exp_models, so might be better to use that for consistency?

Reply:
Ok, makes sense.